ts/features/itwallet/common/components/ItwSkeumorphicCard/ClaimLabel.tsx

```diff
@@ -20,8 +14,8 @@ export type ClaimLabelProps = {
 export const ClaimLabel: React.FunctionComponent<
   React.PropsWithChildren<ClaimLabelProps>
 > = ({
-  fontSize = defaultFontSize,
-  fontWeight = defaultFontWeight,
+  fontSize = 11,
+  fontWeight = "Semibold",
```

Comment:
why did you remove the variables?

Reply:
@Ladirico They are not exported, so they are not used externally, and they are only used once. So why declare the variable in the first place? :)